custom/auth/ante/burntax.go

```diff
@@ -47,11 +51,27 @@ func (btfd BurnTaxFeeDecorator) AnteHandle(ctx sdk.Context, tx sdk.Tx, simulate
 
 	if !simulate {
 		// Compute taxes again.
-		taxes := FilterMsgAndComputeTax(ctx, btfd.treasuryKeeper, msgs...)
+		taxes := FilterMsgAndComputeTax(ctx, btfd.TreasuryKeeper, msgs...)
 
 		// Record tax proceeds
 		if !taxes.IsZero() {
-			err = btfd.bankKeeper.SendCoinsFromModuleToModule(ctx, types.FeeCollectorName, treasury.BurnModuleName, taxes)
+			if currHeight >= TaxPowerSplitHeight {
+				feePool := btfd.DistributionKeeper.GetFeePool(ctx)
+
+				for _, taxCoin := range taxes {
+					splitCoin := sdk.NewCoin(taxCoin.Denom, taxCoin.Amount.Quo(sdk.NewInt(2)))
```

Comment:
`sdk.NewInt(2))` should be a chain param. We can put it into treasury module.

Reply:
Aren't the upgrade handlers still an issue?  Happy new year btw!

Reply:
Happy new year! Yes, if we put ` sdk.NewInt(2))` like a chain param, we need a chain upgrade because it changes the param store.

Reply:
Lets resolve this after we are done with the overall upgrade of the chain. I think we need to wait for @inon-man research to complete before we know when that can happen, so it will be wiser to do this in a later PR and merge in the code as is :)